### DISCUSSION TITLE
Fans in bios need to be in DC mode?

### DISCUSSION THREAD

**Discussion by Grid21** [Rem0o/FanControl.Releases#1088] [Q&A]
My motherboard is an Asus Prime X570-Pro and I noticed a specific bug. Most of my fans are PWM so naturally, it seemed logical to set all the fans to PWM mode in the bios. However, when I went back into my computer login to run FanControl, I was unable to adjust my case fans altogether. But when I go to set the fans back to DC mode in the bios, I am able to regain control of them in the software. Is this normal and to be expected for the fans to be in DC mode to be controlled? Or is there a bug that is causing "PWM mode" to not communicate with FanControl after that mode is changed? Right now at least, with all the fans in DC made, FanControl can talk to and adjust the fans. There does seem to be a slight bug where if I switch the fan to the manual, doesn't matter which one, times FanControl is unable to reinitialize communication control to turn the fan back on. I hope this has made some sense. I'll attach a log file and see if someone can check it out, and help me understand why my fans need to be in DC mode over PWM mode since that's what most of the fans support. 
[log.txt](https://github.com/Rem0o/FanControl.Releases/files/9500167/log.txt)

**Comment by Grid21** [comment]
Hey! I am just following up to see if anyone saw my discussion question regarding needing to put my fans into DC mode in my bios as opposed to using PWM mode. Please see the above post and let me know if this is a bug or if something isn't working correctly.

**Reply by Rem0o** [reply]
I've seen a lot of cases where either PWM or DC would work with FanControl, but not the other.  I can't say much but to just use what works.

**Reply by Grid21** [reply]
I just find it a little silly since all the fans are PWM compatible but FanControl won't control them right until I put them into "DC" mode in the bios. It may be the motherboard itself sending the wrong information, not sure. I thought I'd bring this up for someone to look into for a future bug fix/patch for Asus Prime X570-Pro motherboards.

**Comment by Lev127** [comment]
Have the same problem with Asus x570 Tuf Gaming Plus. If i set my fan mode to PWM Fan Control cannot control them but DC Mode works fine. I assume its a bios version issue or the Mobo itself. I ve had so many weird bugs and issues with this mobo.

**Reply by Raptor959** [reply]
I have this same issue on the same motherboard even with the latest bios
